types/stripe/index.d.ts

```diff
@@ -1045,6 +1045,331 @@ declare namespace Stripe {
+             * document_incomplete, document_invalid, document_manipulated, document_missing_back, document_missing_front,
+             * document_not_readable, document_not_uploaded, document_photo_mismatch, document_too_large, or document_type_not_supported.
+             * A machine-readable code specifying the verification state for this document.
+             */
+            details_code?: string;
+        }
+
+        interface IPersonShared {
+            /**
+             * The account the person is associated with.
+             */
+            account?: string;
+
+            /**
+             * The person’s address.
+             */
+            address?: IAddress;
+
+            /**
+             * The Kana variation of the person’s address (Japan only).
+             */
+            address_kana?: IAddressKana;
+
+            /**
+             * The Kanji variation of the person’s address (Japan only).
+             */
+            address_kanji?: IAddressKanji;
+
+            /**
+             * The person’s date of birth.
+             */
+            dob?: {
+                /**
+                 * The day of birth, between 1 and 31.
+                 */
+                day: number;
+                /**
+                 * The month of birth, between 1 and 12.
+                 */
+                month: number;
+                /**
+                 * The four-digit year of birth.
+                 */
+                year: number;
+            };
+
+            /**
+             * The person’s email address.
+             */
+            email?: string;
+
+            /**
+             * The person’s first name.
+             */
+            first_name?: string;
+
+            /**
+             * The Kana variation of the the person’s first name (Japan only).
+             */
+            first_name_kana?: string;
+
+            /**
+             * The Kanji variation of the person’s first name (Japan only).
+             */
+            first_name_kanji?: string;
+
+            /**
+             * The person’s gender (International regulations require either “male” or “female”).
+             */
+            gender?: 'male' | 'female';
+
+            /**
+             * The person’s last name.
+             */
+            last_name?: string;
+
+            /**
+             * The Kana variation of the person’s last name (Japan only).
+             */
+            last_name_kana?: string;
+
+            /**
+             * The Kanji variation of the person’s last name (Japan only).
+             */
+            last_name_kanji?: string;
+
+            /**
+             * The person’s maiden name.
+             */
+            maiden_name?: string;
+
+            /**
+             * Set of key-value pairs that you can attach to an object.
+             * This can be useful for storing additional information about
+             * the object in a structured format.
+             */
+            metadata?: {
```

Comment:
You should pull this out and use `IOptionsMetadata` in create and update and use `IMetadata` in the object.